### ISSUE TITLE
[Bug]: Transactions table shifts around in UI during reconciliation process

### ISSUE THREAD

**Issue by Cldfire** [actualbudget/actual#1266]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

The reconciliation UI (via the reconcile feature) causes the transaction table to shift around in the UI as you clear / unclear rows:

https://github.com/actualbudget/actual/assets/13814214/6b14675b-ba53-4fd3-81e6-594942f7f751

This has always been possible, but it's happening much more frequently now that the "Create Reconciliation Transaction" button has been added to the UI and takes up a lot of space (forcing the text to wrap much more often).

It's frustrating for the UI to shift during reconciliation. The shifting rows cause you to mentally lose track of your point of reference while reconciling. It also leads to cases where you can accidentally click things you didn't mean to. For example, I want to clear two transactions that were right next to each other in quick succession. Clearing the first transaction caused a table shift, so when I went to click to clear the second the checkbox was no longer where I was clicking, and I ended up _unclearing_ something else.

### Where are you hosting Actual?

Docker

### What browsers are you seeing the problem on?

Safari

### Operating System

Mac OSX

**Comment by j-f1** [comment]
Yeah, that is definitely not ideal, Maybe it would make sense to move “Create Reconciliation Transaction” to an ellipsis menu. What do you think @MatissJanis?

**Comment by MatissJanis** [comment]
I don't think this should be moved away to ellipsis menu.

Instead we could make the reconciliation menu horizontally scrollable for smaller screen sizes.

**Comment by matt-fidd** [comment]
I can't reproduce this now, it's probably still possible in a tiny edge but must be really hard to make happen. The buttons and text scale widths far better than they did a year ago.
